validator/client/validator_propose.go

```diff
@@ -42,6 +43,16 @@ var (
 			"pubkey",
 		},
 	)
+	validatorProposeFailVecSlasher = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "validator",
+			Name:      "failed_proposals_external_slasher",
```

Comment:
please follow prometheus naming standard:
https://prometheus.io/docs/practices/naming/

Reply:
+1

Reply:
changed to 
```prometheus.CounterOpts{
			Name: "validator_proposals_rejected_total",
			Help: "Count the block proposals rejected by protector interface.",
		},
```
```prometheus.CounterOpts{
			Name: "validator_attestations_rejected_total",
			Help: "Count the attestations rejected by protector interface.",
		},
```
is this ok